README.md

```diff
@@ -129,7 +132,7 @@ from dataverse.etl import ETLPipeline
 etl_pipeline = ETLPipeline()
 spark, dataset = etl_pipeline.run(config=ETL_config, verbose=True)
 ```
-ETLPipeline is an object designed to manage the ETL processes. By inserting `ETL_config` which is defined in the previous step into ETLpipeline object and calling the `run` method, stacked ETL blocks are sequentially executed.
+ETLPipeline is an object designed to manage the ETL processes. By inserting `ETL_config` which is defined in the previous step into ETLpipeline object and calling the `run` method, stacked ETL blocks will execute in the order they were stacked.
```

Comment:
@Kimyungi, oh sorry! I misunderstood. Are you asking if it's possible to pass two or more ETL_configs to a single run method? For example, something like` etl_pipeline.run(config=[ETL_config_1, ETL_config_2])?` In that case, actually, only one ETL_config can be passed as the config parameter. Two or more configs cannot be passed simultaneously.

So if a user wants to execute more than one ETL_config, they would need to call run separately for each. It would look like this: first, `etl_pipeline.run(config=ETL_config_1)` and then `etl_pipeline.run(config=ETL_config_2)`.

Given this setup, i think it's quite trivial that ETL_config_2 will run after the completion of ETL_config_1. Tell me if you think this still needs to be written on README!!

Reply:
You are right in meaning the latter. However, in current example, ETL_config is a complete config that loads, processes, and saves datasets. Thus, most users probably do not think that it is possible to run more than one config separately, despite its effectiveness in terms of debugging.

It would be good to keep the example as is, but add simple explanation about it with just one or two sentences.